packages/xo-web/src/common/render-xo-item.js

```diff
@@ -558,7 +558,14 @@ const xoItemToRender = {
   // PIF.
   PIF: pif => (
     <span>
-      <Icon icon='network' color={pif.carrier ? 'text-success' : 'text-danger'} /> {pif.device} ({pif.deviceName})
+      <Icon icon='network' color={pif.carrier ? 'text-success' : 'text-danger'} /> {pif.device} (
+      {pif.vlan === -1
+        ? pif.deviceName
+        : _('keyValue', {
+            key: _('pifVlanLabel'),
+            value: pif.vlan,
+          })}
```

Comment:
Was it in the spec to not show the `deviceName` when we show the VLAN?

Reply:
I was inspired by what exists, and I think that `deviceName` is empty when it's a VLAN.

Reply:
No, that's independent, you can have a VLAN and a `deviceName`.